imports/api/transactions/transaction.js

```diff
@@ -814,6 +815,25 @@ const _genesisTransaction = (userId) => {
   _transact(Meteor.settings.public.Collective._id, userId, rules.VOTES_INITIAL_QUANTITY);
 };
 
+/**
+* @summary reads balance from publicAddress and loads it into user wallet
+* @param {string} userId - id of user being generated within collective
+*/
+const _loadExternalCryptoBalance = (userId) => {
+  const user = Meteor.users.findOne({ _id: userId });
+  if (user.services.metamask != null){
+    user.profile.wallet = _generateWalletAddress(user.profile.wallet);
+    let publicAddress = user.services.metamask.publicAddress
+    let weiBalance = getWeiBalance(publicAddress);
+    
+    user.profile.wallet.currency = 'WEI';
+    user.profile.wallet.balance = weiBalance.toNumber();
```

Comment:
The wallet schema is still expecting numbers. The way I'm thinking about this is that we handle crypto balances in `bigNumbers` for all math operations and convert `toNumber()` only when needed like here. Less than ideal but it seems it's the [current state of handling this issue](https://hackernoon.com/a-note-on-numbers-in-ethereum-and-javascript-3e6ac3b2fad9)